src/System.CommandLine.Tests/Utility/WindowsOnlyFactAttribute.cs

```diff
@@ -4,15 +4,15 @@
 using Microsoft.DotNet.PlatformAbstractions;
 using Xunit;
 
-namespace System.CommandLine.Tests
+namespace System.CommandLine.Tests.Utility
```

Comment:
I typically try to avid the "junk drawer" namespaces like "Utility" because it becomes a grab bag of stuff. Could we move this one into something like "System.CommandLine.Tests.Xunit"

Reply:
Agreed, though I tend to be less concerned about this with test projects. In this case, the child namespaces in the tests all corresponded to child namespaces in System.CommandLine:

![image](https://user-images.githubusercontent.com/547415/79144680-38274f00-7d74-11ea-884f-66411f0e6a05.png)

So my thought was to make this easier to navigate in terms of the tests. Everything that's not tests, I swept into the junk drawer.

But I'm not strongly opinionated either way.

Reply:
(Sometimes I also put folders like this in parentheses, but that just garners funny looks.)

Reply:
Not strongly opinionated either, if we want to stick with the junk drawer i am fine with it